indico/web/templates/forms/person_link_widget.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'forms/base_widget.html' %}
+
+{% macro render_section(id, title=none, placeholder=none) %}
+    {% if title is not none %}
+        <div id="{{ id }}-list-title-{{ field.id }}" class="titled-rule">
+            {{ title }}
+        </div>
+    {% endif %}
+    {% if placeholder is not none %}
+        <div id="no-{{ id }}-placeholder-{{ field.id }}" class="nobody-placeholder">
+            {{ placeholder }}
+        </div>
+    {% endif %}
+    <div id="{{ id }}-list-{{ field.id }}" class="person-list"></div>
+{% endmacro %}
+
+{% block html %}
+    <div id="person-link-field-{{ field.id }}"></div>
+    <input type="hidden" id="{{ field.id }}" name="{{ field.name }}" value="{{ field._value()|tojson|forceescape }}">
+{% endblock %}
+
+{% block javascript %}
+    <script>
+        setupPersonLinkWidget({
+            fieldId: {{ field.id | tojson }},
+            eventId: {{ (field.event.id if field.event else none) | tojson }},
+            roles: {{ field.roles | default([]) | tojson }},
+            sortByLastName: {{ field.sort_by_last_name | default(false) | tojson }},
+            emptyMessage: {{ field.empty_message | default(none) | tojson }},
+            sessionUser: {{ ({
```

Comment:
This is already available in the global `Indico.User`

Reply:
Even though it doesn't support attributes such as `phone` or `address` but it seems perfectly reasonable to support them in a different task.

Reply:
those could be included